lighthouse-core/test/report/report-test.js

```diff
@@ -57,10 +57,26 @@ describe('Report', () => {
     const html = reportGenerator.generateHTML(sampleResults);
 
     assert.ok(html.includes('<footer'), 'no footer tag found');
-    assert.ok(html.includes('printButton = document.querySelector'),
-              'lighthouse-report.js was not inlined');
-    assert.ok(html.includes('.report-body {'), 'report.css was not inlined');
-    assert.ok(!html.includes('&quot;lighthouseVersion'), 'lhresults were not escaped');
+    assert.ok(html.includes('window.lhresults = '), 'report results were inlined');
```

Comment:
I'll check that there's an object on the page, but I don't think we should get in the business of validating the object as it's embedded on the page.